CondCore/CondDB/test/BuildFile.xml

```diff
@@ -17,11 +18,9 @@
 </bin>
 
 <bin   file="testRootStreaming.cpp" name="testRootStreaming">
-  <lib   name="testCondDBDict"/>
```

Comment:
Does SCRAM automatically link the dictionary to everything, or we need to bring back this statement?